efficientdet/det_model_fn.py

```diff
@@ -500,7 +500,9 @@ def _model_outputs(inputs):
     ema = tf.train.ExponentialMovingAverage(
         decay=moving_average_decay, num_updates=global_step)
     ema_vars = utils.get_ema_vars()
-
+  if params['use_horovod']:
```

Comment:
Should you use "params.get('use_horovod', None)",   or you can also add it to hparams_config. Either way is good to me.